src/components/search.tsx

```diff
@@ -14,10 +14,66 @@ import {
   AccordionTrigger,
 } from '@/components/ui/accordion';
 import { logAnalytics } from '@/api/analytics';
+import { sortByFilterOptions } from '@/constants/sortByFilterOptions';
 
 type DecisionBodyFilterProps = {
   decisionBodies: Record<string, DecisionBody>;
 };
+
+export function SortDropdown() {
+  const {
+    searchOptions: { sortBy, sortDirection },
+    setSearchOptions,
+  } = useSearch();
+
+  // Use sortByFilterOptions instead of sortByOptions
+  const options = useMemo(
+    () =>
+      Object.values(sortByFilterOptions).map((opt) => ({
```

Comment:
since `sortByFilterOptions` is now an array, you can just use `map` on it directly:

```suggestion
      sortByFilterOptions.map((opt) => ({
```